README.md

```diff
@@ -1,104 +1,130 @@
 
-- SDK does not support policy revocation.
-- We expect breaking changes.
+## 1. Install nucypher-ts
 
-## Supported networks
+We'll kick things off by installing `nucypher-ts` – a TypeScript library for employing access control functionality in the browser. The APIs for leveraging TAC functionality are contained in nucypher-ts.
 
-`nucypher-ts` is available on the following networks:
+```
+yarn add @nucypher/nucypher-ts
+```
+
+## 2. Build a Cohort
 
-- Polygon
-- Mumbai (Polygon testnet)
+Next, we'll parametrize a _Cohort_ to correspond to our risk preferences. Cohort objects delineate the group of independent nodes that will collectively provide access control service to a given data sharing flow. _Threshold_ and _Shares_ are two parameters used to construct a Cohort. For example, a `3-of-5` Cohort requires responses – the delivery of shares – from a minimum of 3 out of a total of 5 Cohort members in order to reconstruct the original plaintext data.
 
-## Usage
+We generate a Cohort object by:
 
-Run with:
+```javascript
+import { Cohort } from '@nucypher/nucypher-ts';
 
-```bash
-yarn install
-yarn test
-yarn build
+const config = {
+  threshold: 3,
+  shares: 5,
+  porterUri: 'https://porter-tapir.nucypher.community',
+};
+const newCohort = await Cohort.create(config);
 ```
 
-## Development
+Notice that we have also provided a `porterUri`. **Porter is a web-based service that interacts with nodes on the network on behalf of applications – an "Infura for TAC".** In this example, we've chosen a `tapir` (testnet) [Porter](https://github.com/nucypher/nucypher-porter) endpoint.
 
-Install git hooks
+## 3. Create Conditions
 
-```bash
-npx husky install
-```
+We will now specify the conditions on which data access will be predicated – i.e. what will the data requester need to prove in order to gain decryption rights. In this tutorial, nodes will check that the requester owns a specific ERC721 NFT:
 
-Generate contract typings
+```javascript
+import { Conditions } from '@nucypher/nucypher-ts';
 
-```bash
-yarn typechain
+const NFTOwnership = new Conditions.ERC721Ownership({
+  contractAddress: '0xBC4CA0EdA7647A8aB7C2061c2E118A18a936f13D',
+  chain: 5, // tapir uses the Goerli testnet
+  parameters: [5954],
+});
 ```
 
-Prepare a new release
+Note that there are other Condition [types](https://docs.threshold.network/app-development/threshold-access-control-tac/references/conditions), and it is possible to compose and combine multiple Condition objects into a [_ConditionSet_](https://docs.threshold.network/app-development/threshold-access-control-tac/references/condition-set)_:_
```

Comment:
```suggestion
Note that `ERC721Ownership` above serves as a wrapper function. The ERC 721 `ownerOf` contract method is what actually finds a `tokenID` and returns the owner's address, such that it can be compared with the requestor's signature. For help customizing the wrapper function, see the [References](https://docs.threshold.network/app-development/threshold-access-control-tac/references/conditions#conditions.erc721ownership) section of the full documentation. 

Note that there are other Condition [types](https://docs.threshold.network/app-development/threshold-access-control-tac/references/conditions), and it is possible to compose and combine multiple Condition objects into a [_ConditionSet_](https://docs.threshold.network/app-development/threshold-access-control-tac/references/condition-set)_:_
```